locksmith/src/operations/userOperations.ts

```diff
@@ -0,0 +1,41 @@
+const ethJsUtil = require('ethereumjs-util')
+const models = require('../models')
+const { User, UserReference } = models
+import * as types from '../types'
+
+class UserOperations {
```

Comment:
I actually intentionally did not create classes for the other operations. Could you expand on why you think they're required here?

Reply:
this was/is me reacting to some typescript things or at least the way that I understand them.  Our js approach written in typescript puts the methods in the global scope, there may be a better way to approach this

Reply:
Maybe you're looking for namespaces? https://www.typescriptlang.org/docs/handbook/namespaces.html

Alternatively, just declaring modules.